### ISSUE TITLE
ci/cd: experimental labeler

### ISSUE THREAD

**Issue by kamilsk** [octomation/go-module#34]
see
- https://github.com/spf13/cobra/blob/master/.github/labeler.yml
- https://github.com/spf13/cobra/blob/master/.github/workflows/labeler.yml

**Comment by kamilsk** [comment]
blocked by [#33](https://github.com/octomation/go-module/issues/33)

**Comment by kamilsk** [comment]
won't do
